stripe-ui-core/res/values/strings.xml

```diff
@@ -4,4 +4,10 @@
   <string name="address_label_phone_number">Phone number</string>
   <!-- Error description for incomplete phone number -->
   <string name="incomplete_phone_number">Incomplete phone number</string>
+  <!-- Used for accessibility to indicate that when you press the dropdown arrow on a form (such as a country or bank), it will bring up a menu to allow you to change the value. -->
+  <string name="change">Change</string>
+  <!-- The label of a text field that is optional. For example, 'Email (optional)' or 'Name (optional) -->
+  <string name="form_label_optional">%s (optional)</string>
+  <!-- Used when a required field is blank and submit is clicked. -->
+  <string name="blank_and_required">This field cannot be blank.</string>
```

Comment:
TODO - move these in Lokalise and pull strings.

Reply:
FYI for the ones not translated we usually put them in [totranslate.xml](https://github.com/stripe/stripe-android/blob/2afde7200db52aa49370681e984a9aba021fcdc3/link/res/values/totranslate.xml#L2) first